src/settings/components/AnonymizingTypeSelect/AnonymizingTypeSelectContainer.js

```diff
@@ -87,10 +91,19 @@ class AnonymizingTypeSelectContainer extends Component {
     } = this.props;
 
     return (
-      <AnonymizingTypeSelect
-        name={path}
-        types={this.getClosingTypes(name, path)}
-      />
+      <Fragment>
+        <AnonymizingTypeSelect
+          name={path}
+          types={this.getClosingTypes(name, path)}
+        />
+        <br />
```

Comment:
Are you sure that we need to add br instead of CSS settings?

Reply:
Thanks for the comment. That's a good point.